packages/wdio-browserstack-service/src/service.ts

```diff
@@ -67,6 +71,10 @@ export default class BrowserstackService implements Services.ServiceInstance {
         if (strict) {
             this._failureStatuses.push('pending')
         }
+
+        if (((_caps as any)['wdio:cid'] as string) === process.env.BEST_PLATFORM_CID) {
```

Comment:
Let's handle caps properly and don't cast to any

Reply:
Fixed the approach here to use `process.env.WDIO_WORKER_ID` instead of storing cid in caps